test/wasmBenchmarker/benchmark.py

```diff
@@ -140,135 +142,133 @@ def compile_tests(emcc_path, path, only_game=False, only_simd=False, compile_any
 
         test_names.append(name)
 
-        if not compile_anyway and os.path.exists(path + "/wasm/" + name + ".wasm"):
-            if (verbose): print(name + ".wasm is found; compilation skipped")
+        if not compile_anyway and os.path.exists(f"{path}/wasm/{name}.wasm"):
+            if (verbose): print(f"{name}.wasm is found; compilation skipped")
             continue
 
-        extraFlags = "-msimd128" if file.startswith("simd") else ""
-
-        if (verbose): print("compiling " + name)
-        bob_the_stringbuilder = emcc_path + " " + path + "/" + file + " " + extraFlags + " -s WASM=1 -s EXPORTED_FUNCTIONS=_runtime -s EXPORTED_RUNTIME_METHODS=ccall,cwrap -o " + path + "/wasm/" + name + ".wasm"
-        if verbose: print(bob_the_stringbuilder)
-        os.system(bob_the_stringbuilder)
+        flags = ("-msimd128" if file.startswith("simd") else "") \
+              + " -s WASM=1 -s EXPORTED_FUNCTIONS=_runtime"      \
```

Comment:
Do you need \ if you put the + before the next line? Is this a common style?

Reply:
I've found these two:
```python
long_string = "This is a very long string " + \
              "that spans multiple lines " + \
              "but appears as a single line."
```
```python
long_string = ("This is a very long string "
               "that spans multiple lines "
               "but appears as a single line.")
```

In the first case, the place of **+** is personal preference, in my humble opinion.

Reply:
The second looks the best. There is no \ or +

Reply:
Okay. I'll change it.